WORKSPACE

```diff
@@ -14,15 +14,122 @@
 
 workspace(name = "com_github_googleapis_google_cloud_cpp_spanner")
 
+###
```

Comment:
Shouldn't this be in the google_cloud_spanner_deps.bzl file too?  Yes, it is temporary, but we want that file to load all the necessary dependencies, right?

Reply:
I'll refactor this in a follow on PR.